src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -493,19 +493,6 @@
         <PackageVersion>$(PackageVersion)</PackageVersion>
       </PackageFile>
 
-      <!-- Include Sources. Deduplicate so the nuspec doesn't contain multiple entries for each source file. -->
-      <PackageSources Include="@(File)"
-                      KeepMetadata="TargetPath;IsSourceCodeFile"
-                      KeepDuplicates="false"
-                      Condition="'%(File.IsSourceCodeFile)'=='true'" />
-      <!-- Add a placeholder source file if there are symbols but no sources. -->
```

Comment:
Interesting, that this part was no longer needed.  I guess NuGet fixed this bug @dagood.  @wtgodbe can you double check that we still get the symbols?

Reply:
Actually looks like this was an assumption here: https://github.com/dotnet/buildtools/blob/3306ca2481414624a56f22e0ee28f7c08b93037e/src/Microsoft.DotNet.Build.Tasks.Packaging/src/NuGetPack.cs#L264.  I'm a little surprised if that no longer holds.

Reply:
We have decided that our symbol packages are only used as a transport mechanism so they don't need to be "valid" by that sense.

Reply:
I just compared the symbol packages before and after this change, other than the lack of source files, they look identical. I can share a new version of the packages if you guys would like to inspect them yourselves.

Reply:
Good spot, this actually is probably still needed. The impact is MyGet: symbol packages are only barely a NuGet feature. I don't think this is a bug, it's just how MyGet classifies the packages. Depending on how the build pushes symbol packages to MyGet not getting this right can cause issues like symbol packages showing up as the "main" package for a certain id/version. (There is a symbol-package-specific feed but I don't think it's reliably used.) The opposite is also true, like in https://github.com/dotnet/coreclr/issues/13798, where something intended as a "normal" package is treated like a symbol package.

Reply:
@dagood didn't we stop pushing nuget packages to myget?

Reply:
We don't use MyGet as the *only* way to transport nupkgs from one of our builds to another, but it's still a maintained public endpoint and it's still used by a bunch of builds in the dotnet org.

I would like to simply go through and fix this, but I don't think it's best to start by removing this placeholder source file. The problems it can cause on the MyGet feed aren't easy to detect, and can have downstream effects that may be flaky and I don't think would be easy to diagnose.

Reply:
What is suggestion then @dagood? Should we abandon this PR?

Reply:
I think it's mainly fine, we just need to keep making the placeholder file in some way. Maybe just keep:

```xml
<PackageSources Include="$(PlaceholderFile)" Condition="'@(PackageSymbolFiles)'!='' AND '@(PackageSources)'==''">
  <IsSourceCodeFile>true</IsSourceCodeFile>
  <TargetPath>src</TargetPath>
</PackageSources>
<PackageFile Include="@(PackageSources)" />
```

Or reimplement in `NuGetPack.cs` maybe, seems like that might be cleaner.

Reply:
I'm ok either way.  Moving it into NuGetPack seems like a separate issue.

Reply:
OK yeah for this change lets just keep the placeholder with an updated comment.

Reply:
I've just updated this to add the placeholder back